gpu-direct/rhel/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM registry.access.redhat.com/ubi8:latest
```

Comment:
This image generated by this Dockerfile will work only on openshift right ?

maybe name it : `Dockerfile.openshift` ?

Reply:
I don't think there's much preventing the RHEL images being deployed on k8s?
Also there's no other Dockerfile in this directory that would lead to confusion (and thus differentiating with `.openshift`)?